service/actions/errors.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {ErrorTypes} from 'service/constants';
+
+export function dismissErrorObject(index) {
+    return {
+        type: ErrorTypes.DISMISS_ERROR,
+        displayable: false,
+        index
+    };
+}
+
+export function dismissError(index) {
+    return async (dispatch) => {
+        dispatch(dismissErrorObject(index));
+    };
+}
+
+export function generalErrorObject(error, displayable = false) {
```

Comment:
Is there a better name that you could use for this? It's weird that the function name doesn't start with a verb